indico/web/client/styles/partials/_forms.scss

```diff
@@ -66,7 +66,7 @@ form {
 
     .form-field {
       float: left;
-      width: 80%;
+      width: 85%;
```

Comment:
This change is break forms pretty much everywhere:

![image](https://user-images.githubusercontent.com/179599/123667977-f086c900-d83a-11eb-8628-e7b7e4f5684b.png)

![image](https://user-images.githubusercontent.com/179599/123668050-009ea880-d83b-11eb-92c7-f9c313700e4a.png)

![image](https://user-images.githubusercontent.com/179599/123668074-085e4d00-d83b-11eb-8af6-ee97726ec7c6.png)

Reply:
I reverted this for this PR as it is unrelated and I'll fix it in a separate commit.